string alert_message
int32 severity
